test/models/AppModel/test.js

```diff
@@ -0,0 +1,131 @@
+          await AppFactory.Create();
+        }
+      });
+      it('Should check that getAll returns correct number of records', async () => {
+        result = await AppModel.getAll();
+        expect(result.apps.length).to.be.eq(appsCount);
+      });
+    });
+    describe('On error case', async () => {
+      beforeEach(async () => {
+        result = await AppModel.getAll();
+      });
+      it('Should check that the error exists', async () => {
+        expect(result.error).is.exist;
+      });
+      it('Should return error message', async () => {
+        expect(result.error.message).to.be.eq('App not found!');
+      });
+    });
+  });
+  describe('On aggregate', async () => {
+    let name, admins, blacklists, app, aggResult, result;
+    beforeEach(async () => {
+      await dropDatabase();
+      name = faker.name.firstName();
+      admins = [faker.name.firstName().toLowerCase()];
+      blacklists = [faker.name.firstName().toLowerCase()];
+      app = await AppFactory.Create({
+        name, admins, blacklists,
+      });
+    });
+    it('Should return from our request: id, name and admins. Using group stage', async () => {
+      aggResult = await AppModel.aggregate([
+        {
+          $group: {
+            _id: '$_id',
+            name: { $first: '$name' },
+            admins: { $first: '$admins' },
+          },
+        },
+      ]);
+      [result] = aggResult.result;
+      expect(result).to.deep.eq(_.pick(app, ['_id', 'name', 'admins']));
+    });
+    it('Should return from our request: name and admins. Using match and project stage', async () => {
+      aggResult = await AppModel.aggregate([
+        { $match: { name: app.name } },
+        {
+          $project: {
+            _id: 0, name: 1, admins: 1,
+          },
+        },
+      ]);
+      [result] = aggResult.result;
+      expect(result).to.deep.eq(_.pick(app, ['name', 'admins']));
+    });
+    it('Should return from our request: whole app. Using match stage', async () => {
+      aggResult = await AppModel.aggregate([
+        { $match: { name: app.name } },
+      ]);
+      [result] = aggResult.result;
+      expect(result).to.deep.eq(app._doc);
+    });
+    it('Should return undefined, when aggregate has not result', async () => {
+      aggResult = await AppModel.aggregate([
+        { $match: { surname: app.name } },
+        {
+          $project: {
+            _id: 0, name: 1, admins: 1,
+          },
+        },
+      ]);
+      [result] = aggResult.result;
+      expect(result).to.be.undefined;
+    });
+  });
+  describe('On updateOne', async () => {
+    let name, app, result;
+    beforeEach(async () => {
+      name = faker.name.firstName();
+      app = await AppFactory.Create({ name });
+    });
+    it('Should update name at request and get true', async () => {
+      result = await AppModel.updateOne({ name, updData: { $set: { name: faker.name.firstName() } } });
+      expect(result.result).to.be.true;
```

Comment:
You can use destructurization (const {result} = ...) then in expect you dont need to result.result, just result